src/main/Catalog.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+
+function Catalog() {
+  let data;
+  const [state, setstate] = useState([]);
+  let xhttp = new XMLHttpRequest();
+  xhttp.onload = function () {
+    if (this.readyState === 4 && this.status === 200) {
+      data = JSON.parse(xhttp.responseText);
+    }
+    setstate(data);
+  };
+  xhttp.open("GET", "https://api.punkapi.com/v2/beers", true);
+  xhttp.send(null);
+
+  return (
+    <div className="catalog">
+      {state.map((obj) => {
```

Comment:
_Optional:_ you can consider using object destructuring in order to get rid of `obj.` everywhere.
Also, let's replace `obj` with something more specific.